chainntnfs/txnotifier.go

```diff
@@ -1009,8 +1009,8 @@ func (n *TxNotifier) RegisterSpend(outpoint *wire.OutPoint, pkScript []byte,
 	if err == nil {
 		if hint > startHeight {
 			Log.Debugf("Using height hint %d retrieved from cache "+
-				"for %v instead of %d", hint, ntfn.SpendRequest,
```

Comment:
same here

Reply:
Downgraded.

Reply:
I don't think the `Debug` should be downgraded, it has been pretty helpful in the past when users encounter weird chainntnfs issues. I meant that I thought the log line shouldn't be changed in this pull given it's supposed to be fixing something in `discovery` and I have no problem reviewing the change in another pull